models/evc.py

```diff
@@ -781,27 +781,29 @@ def remove_path_flows(
             uni_flows = {}
 
         for dpid, flows in uni_flows.items():
-            dpid_flows_match.setdefault(dpid, [])
             for flow in flows:
-                dpid_flows_match[dpid].append({
+                flow_mod = {
                     "cookie": flow["cookie"],
                     "match": flow["match"],
                     "owner": "mef_eline",
                     "cookie_mask": int(0xffffffffffffffff)
-                })
+                }
+                dpid_flows_match[dpid]["flows"].append(flow_mod)
+                out_flows[dpid].append(flow_mod)
+
+        try:
+            self._send_flow_mods(
+                dpid_flows_match, 'delete', force=force, by_switch=True
+            )
+        except FlowModException as err:
+            log.error(f"Error deleting path, {err}")
```

Comment:
Maybe in the log.error here in the meantime would be helpful to add the `{path}` since `remove_path_flows` is generic? Understood that with the error handling this will be much nicer, but I'm still a bit concerned if this log error happens it'll be a bit obscure to debug and understand.

Reply:
Added `{path}` to the log.